web_poet/testing/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class FieldMissing(AssertionError):
```

Comment:
This one curently returns extra stracktraces unlike the other ones.

Reply:
Fixed in https://github.com/scrapinghub/web-poet/pull/129/commits/3bc857ad905419d573cbfbc0afdd6435ca2c2668